Headers/DebugServer2/Architecture/ARM/HardwareBreakpointManager.h

```diff
@@ -23,25 +23,25 @@ class HardwareBreakpointManager : public BreakpointManager {
   ~HardwareBreakpointManager() override;
 
 public:
-  virtual ErrorCode add(Address const &address, Type type, size_t size,
-                        Mode mode) override {
+  ErrorCode add(Address const &address, Type type, size_t size,
+                Mode mode) override {
     return kErrorUnsupported;
   };
 
 protected:
-  virtual bool hit(Target::Thread *thread) override { return false; };
+  bool hit(Target::Thread *thread) override { return false; };
 
 protected:
-  virtual ErrorCode enableLocation(Site const &site) override {
+  ErrorCode enableLocation(Site const &site) override {
     return kErrorUnsupported;
   };
 
-  virtual ErrorCode disableLocation(Site const &site) override {
+  ErrorCode disableLocation(Site const &site) override {
     return kErrorUnsupported;
   };
 
 public:
-  virtual int maxWatchpoints();
+  int maxWatchpoints();
```

Comment:
This one you should leave as `virtual` because if the ARM HardwareBreakpointManager is subclassed, overriding `maxWatchpoints()` is going to lead to some nice bugs.